gui/slick/interfaces/default/config_notifications.tmpl

```diff
@@ -1544,13 +1551,13 @@
                             <div id="content_trakt_use_rolling_download">
               	                <div class="field-pair">
                                     <label for="trakt_rolling_num_ep">
-                                        <span class="component-title">Number of Episode:</span>
+                                        <span class="component-title">Quantity of episodes:</span>
```

Comment:
@gborri @fernandog You don't say "Quantity of episodes" in English, "Number of episodes" is better